### PR TITLE
Mkdocs 0.13

### PR DESCRIPTION
**Pull request by d0ugal**
I added commit messages to explain each change, GitHub partially hides this, so I'll comment here too...

### CODE REVIEW

File: readthedocs/doc_builder/backends/mkdocs.py [hunk 1]
```diff
@@ -9,10 +7,7 @@
 from django.template import Context, loader as template_loader
 
 from doc_builder.base import BaseBuilder, restoring_chdir
-from search.utils import parse_content_from_file, parse_headers_from_file, parse_sections_from_file
 from projects.utils import run
-from projects.constants import LOG_TEMPLATE
-from tastyapi import apiv2
```
Comment (d0ugal) [inline readthedocs/doc_builder/backends/mkdocs.py]: All of the above are unused imports.

File: readthedocs/doc_builder/backends/mkdocs.py [hunk 2]
```diff
@@ -81,22 +76,6 @@ def append_conf(self, **kwargs):
...
-                            root_path = root.replace(docs_dir, '')
-                            full_path = os.path.join(root_path, filename.lstrip('/')).lstrip('/')
-                        else:
-                            if root == '.':
-                                root_path = ''
-                            else:
-                                root_path = re.sub('^./', '', root)
-                            full_path = os.path.join(root_path, filename.lstrip('/')).lstrip('/')
-                        user_config['pages'].append([full_path])
-
```
Comment (d0ugal) [inline readthedocs/doc_builder/backends/mkdocs.py]: MkDocs automatically includes all Markdown files now if the user doesn't manually list them. So this is just duplication.

File: readthedocs/templates/mkdocs/readthedocs/versions.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
```
Comment (d0ugal) [inline readthedocs/templates/mkdocs/readthedocs/versions.html]: This is the only customisation needed in ReadTheDocs, so I made it easier but rolling a [small MkDocs release](http://www.mkdocs.org/about/release-notes/#version-0132-2015-05-30). See: https://github.com/mkdocs/mkdocs/pull/572

**Comment by d0ugal** [comment]
I've tested this only with the MkDocs documentation. Feedback very welcome. Is there a handy way I can get a list of MkDocs projects that I can add locally and test they all build?

**Comment by ericholscher** [comment]
Yea, I can get something to you in the next couple days. Running out to the
farmers market now, but definitely excited to see this land.

A lot of what we were doing was hacking around mkdocs bugs, so as you
pointed out, there's lots of stuff that can likely be removed now :)

On Sat, May 30, 2015 at 10:14 AM, Dougal Matthews <notifications@github.com>
wrote:

> I've tested this only with the MkDocs documentation. Feedback very
> welcome. Is there a handy way I can get a list of MkDocs projects that I
> can add locally and test they all build?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtfd/readthedocs.org/pull/1305#issuecomment-107067238>
> .
>



-- 
Eric Holscher
Maker of the internet residing in Portland, Oregon
http://ericholscher.com

**Comment by shaoner** [comment]
Hi,

Thanks for the update!
This should work with 0.13: https://github.com/shaoner/ziti (http://ziti.readthedocs.org)

**Comment by ericholscher** [comment]
Well, we have over 2000 mkdocs projects at this point, so I don't know about hand testing them all :) I imagine a good number of them are test projects that folks imported and are just using our mkdocs integration, and likely don't even have a mkdocs.yml.

http://sequelize.readthedocs.org/en/latest/ is probably a good one to test, as they're one of our biggest mkdocs users.

**Comment by ericholscher** [comment]
Another good one: http://osquery.readthedocs.org/en/stable/

**Comment by d0ugal** [comment]
Great. thanks! I'll start by testing those and see how we get on it. I'd be interested to see other big MkDocs users as I don't have many ways of finding out otherwise!

**Comment by ericholscher** [comment]
Went ahead and merged, and it looks good on initial inspection.

Nice that the RTD footer actually docks in now :)

**Comment by d0ugal** [comment]
w00t!

/me waits for people to should at me for breaking stuff.

**Comment by shaoner** [comment]
@d0ugal works great for me!

**Pull request merged**